tests/test_fetch_ndc_database.py

```diff
@@ -0,0 +1,83 @@
+import io
+import os.path
+import zipfile
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from fdadb.management.commands import fetch_ndc_database
+from fdadb.models import MedicationName, MedicationNDC, MedicationStrength
+
+THIS_FILE_DIR = os.path.dirname(os.path.realpath(__file__))
+TEST_PRODUCT_FILE = os.path.join(THIS_FILE_DIR, "test_data", "product.txt")
+
+
+def as_zip_file(names_and_contents):
+    zip_buffer = io.BytesIO()
+    with zipfile.ZipFile(zip_buffer, "a", zipfile.ZIP_DEFLATED) as zip_file:
+        for file_name, data in names_and_contents:
+            zip_file.writestr(file_name, data)
+    return zip_buffer.getvalue()
+
+
+def fake_database_file():
+    with open(TEST_PRODUCT_FILE, "rb") as f:
+        return as_zip_file([("product.txt", f.read())])
+
+
+class FetchNdcDatabaseTestCase(TestCase):
+    @mock.patch.object(fetch_ndc_database.Command, "fetch_database_file")
+    def test_command(self, fetch_database_file):
+        fetch_database_file.return_value = fake_database_file()
+
+        for model in (MedicationName, MedicationNDC, MedicationStrength):
+            self.assertEqual(model.objects.count(), 0)
+
+        call_command(fetch_ndc_database.Command())
```

Comment:
Why not just `call_command("fetch_ndc_database")`?

Reply:
Because I already had the module imported, used `Command` above, and I tend to use variable/constant rather than literals to avoid typos.
But in the case of `call_command` you're right, it reads better with the literal.